libp2p/pubsub/pubsub.py

```diff
@@ -154,7 +157,13 @@ def get_hello_packet(self) -> rpc_pb2.RPC:
         peer_id = stream.mplex_conn.peer_id
 
         while True:
-            incoming: bytes = await read_varint_prefixed_bytes(stream)
+            try:
+                incoming: bytes = await read_varint_prefixed_bytes(stream)
+            except (ParseError, IncompleteReadError) as error:
+                logger.debug(
+                    "read corrupted data from peer %s, error=%s", peer_id, error
+                )
+                continue
```

Comment:
With either case mentioned above, we should return from the loop.
https://github.com/libp2p/go-libp2p-pubsub/blob/master/comm.go#L38-L46